src/Razor/src/Microsoft.AspNetCore.Razor.ExternalAccess.LegacyEditor/InternalAPI.Unshipped.txt

```diff
@@ -1 +1,3 @@
 ﻿#nullable enable
+static Microsoft.AspNetCore.Razor.ExternalAccess.LegacyEditor.RazorWrapperFactory.GetWrappedTagHelperFactsService() -> Microsoft.AspNetCore.Razor.ExternalAccess.LegacyEditor.IRazorTagHelperFactsService!
+*REMOVED*static Microsoft.AspNetCore.Razor.ExternalAccess.LegacyEditor.RazorWrapperFactory.WrapTagHelperFactsService(object! obj) -> Microsoft.AspNetCore.Razor.ExternalAccess.LegacyEditor.IRazorTagHelperFactsService!
```

Comment:
ℹ️ This change is OK. We reserve the right to change `RazorWrapperFactory` in the legacy editor shim, even though it is accessible to Web Tools. This class is not used directly by Web Tools and is even marked as `EditorBrowsableState.Never` to hide it from IntelliSense.